pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -33,15 +33,22 @@ import (
 )
 
 var (
-	logger = capnslog.NewPackageLogger("github.com/rook/rook", "ceph-csi")
+	logger      = capnslog.NewPackageLogger("github.com/rook/rook", "ceph-csi")
+	ConfigMutex sync.Mutex
```

Comment:
Can this be renamed or is it still used outside the cluster?
```suggestion
	configMutex sync.Mutex
```